style.css

```diff
@@ -0,0 +1,92 @@
+*,
+*::after,
+*::before {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: url(images/header-illsutration-mobile.png);
+  background-color: white;
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: top 2.75rem right -0.5rem;
+}
```

Comment:
- [ ] Please fix your background image I recommended please use this image in the `hero-section` class because this is only for that particular section also stick to the Figma design your background image is not the same as the Figma design  so  kindly fix these issues :+1:  check below :point_down: the comparison between your design and Figma's design :slightly_smiling_face: 

**Your Design**
![Screenshot from 2023-06-07 23-48-32](https://github.com/MoliBravo/Portfolio_setup_and_mobile_first/assets/45226848/7a7b9772-7705-450b-8b09-4fa3751de8c8)

**Figma design**
![Screenshot from 2023-06-07 23-48-44](https://github.com/MoliBravo/Portfolio_setup_and_mobile_first/assets/45226848/bcf2c69e-0a53-4ded-93ac-d2eb0234aefa)